tests/interfaces-network/task.yaml

```diff
@@ -0,0 +1,85 @@
+summary: |
+  The network interface allows a snap to access the network as a client.
+
+  A snap which defines the network plug must be shown in the interfaces list.
+  The plug must be autoconnected on install and, as usual, must be able to be
+  reconnected.
+
+  A snap declaring a plug on this interface must be able to access network services.
+
+environment:
+  SNAP_NAME: network-consumer
+  SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+  PLUG: network
```

Comment:
Can we please have the word "network" inlined instead of this variable?  This is a constant really, and is used in contexts which are not about a plug, specifically. So it's obscuring the test rather than helping out.